record/index.html.in

```diff
@@ -39,7 +39,7 @@
                         Related Records
                     </strong>
                 </a>&nbsp;&nbsp;
-                <a id="permalink" ng-href="{{::ctrl.permalink()}}" uib-tooltip="This link stores your search criteria as a URL. Right click and save." tooltip-placement="bottom">
+                <a id="permalink" ng-href="{{::ctrl.permalink()}}" uib-tooltip={{::tooltip.permalink}} tooltip-placement="bottom">
```

Comment:
Because of [this](https://github.com/informatics-isi-edu/chaise/blob/master/record/index.html.in#L9) (using controller as syntax) and after attaching `tooltip` to vm, this can be changed to `ctrl.tooltip.permalink`.